cincinnati/src/common.rs

```diff
@@ -0,0 +1,8 @@
+/// Adds "/" if the input is not empty
```

Comment:
I think the actual logic you want is something like "make sure any (optional) namespace starts with a slash and ends without one".

Reply:
See above, I relied on the user to provide a correct string as this is only a compatibility feature.

Reply:
Ack.